@here/harp-mapview/lib/MapView.ts

```diff
@@ -807,16 +806,18 @@ export class MapView extends THREE.EventDispatcher {
             ConcurrentDecoderFacade.defaultWorkerCount = this.m_options.decoderCount;
         }
 
-        this.m_visibleTileSetOptions = { ...MapViewDefaults };
+        this.m_visibleTileSetOptions = {
+            ...MapViewDefaults,
+            clipPlanesEvaluator:
+                options.clipPlanesEvaluator !== undefined
+                    ? options.clipPlanesEvaluator
+                    : new TiltViewClipPlanesEvaluator()
+        };
 
         if (options.projection !== undefined) {
             this.m_visibleTileSetOptions.projection = options.projection;
         }
 
-        if (options.clipPlanesEvaluator !== undefined) {
-            this.m_visibleTileSetOptions.clipPlanesEvaluator = options.clipPlanesEvaluator;
-        }
-
```

Comment:
Why it is removed? This way we disallow users to specify it via options...

Reply:
It's still possible, look few lines above.